### ISSUE TITLE
RPM installation failure - usermod: user riakcs is currently logged in

### ISSUE THREAD

**Issue by aggress** [basho/node_package#64]
Centos 6.4
rpm -ivh riak-cs-control-1.0.0-1.el6.x86_64.rpm
Preparing...                ########################################### [100%]
usermod: user riakcs is currently logged in
error: %pre(riak-cs-control-1.0.0-1.el6.x86_64) scriptlet failed, exit status 8
error:   install: %pre scriptlet failed (2), skipping riak-cs-control-1.0.0-1.el6

**Comment by hectcastro** [comment]
Should these steps have reproduced that?

``` bash
[vagrant@riak1 ~]$ uname -a
Linux riak1 2.6.32-358.el6.x86_64 #1 SMP Fri Feb 22 00:31:26 UTC 2013 x86_64 x86_64 x86_64 GNU/Linux
[vagrant@riak1 ~]$ sudo rpm -ivh riak-cs-1.3.1-1.el6.x86_64.rpm
Preparing...                ########################################### [100%]
   1:riak-cs                ########################################### [100%]
[vagrant@riak1 ~]$ sudo rpm -ivh stanchion-1.3.1.1-1.el6.x86_64.rpm
Preparing...                ########################################### [100%]
   1:stanchion              ########################################### [100%]
[vagrant@riak1 ~]$ sudo rpm -ivh riak-cs-control-1.0.0-1.el6.x86_64.rpm
Preparing...                ########################################### [100%]
   1:riak-cs-control        ########################################### [100%]
[vagrant@riak1 ~]$
```

**Comment by jaredmorrow** [comment]
No.  You are installing everything at once which is what I tested and that works.  The bug is hit if the user sets up riak-cs and/or stanchion and is running it, then decides later to install cs-control.  This then tries to install the package which tries to modify the the user and that is what fails.

**Comment by hectcastro** [comment]
Ah. Was trying to figure out why I hadn't hit this with the Vagrant + CS project, and I think it's because Chef defers the service starts/restarts until the end of the run.

**Comment by jaredmorrow** [comment]
Looks like a user on riak-users hit this during upgrade.  Now that I have more details I'll fix the usermod line.

**Comment by jaredmorrow** [comment]
Fixed in #75 

Tested by leaving riak running, uninstalling and installing while riak is currently logged in.

```
riak     11202  0.0  0.0  11496   328 ?        S    15:43   0:00 /usr/lib64/riak/erts-5.9.1/bin/epmd -daemon
riak     19782  0.0  0.0  13572   712 ?        S    17:36   0:00 /usr/lib64/riak/erts-5.9.1/bin/run_erl -daemon 
riak     19950  0.0  0.0 112096  1048 ?        Ss   17:36   0:00      \_ sh -s disksup
riak     19951  0.0  0.0   4248   556 ?        Ss   17:36   0:00      \_ /usr/lib64/riak/lib/os_mon-2.2.9/priv/bin/memsup
riak     19953  0.0  0.0   4244   308 ?        Ss   17:36   0:00      \_ /usr/lib64/riak/lib/os_mon-2.2.9/priv/bin/cpu_sup


[buildbot@build-fedora-17-64 packages]$ sudo rpm -ivh riak-1.4.0.b5d61c75-1.fc17.x86_64.rpm
Preparing...                ########################################### [100%]
usermod: no changes
   1:riak                   ########################################### [100%]
```
